src/engine/client/display.c

```diff
@@ -649,24 +648,39 @@ void Display_SetupDraw(double delta) {
     }
     plClearBuffers(clear_flags);
 
+    plBindFrameBuffer(game_target, PL_FRAMEBUFFER_DRAW);
     plSetupCamera(g_state.camera);
 }
 
 void Display_DrawScene(void) {
+    plSetShaderProgram(programs[SHADER_DEFAULT]);
     //Map_Draw();
     //DrawActors(cur_delta);
     DrawParticles(cur_delta);
 }
 
 void Display_DrawInterface(void) {
+    plSetShaderProgram(programs[SHADER_DEFAULT]);
     SetupFrontendCamera(640, 480);
     FE_Draw();
 
-    SetupFrontendCamera(0, 0);
+    SetupFrontendCamera(640, 480);
```

Comment:
Console and ImGui should render at full size of the viewport, not 640x480 like the frontend.